packages/models-library/src/models_library/projects.py

```diff
@@ -200,6 +200,7 @@ class Config:
 
 
 class Owner(BaseModel):
+    user_id: conint(ge=1) = Field(..., description="Owner's identifier when registered in the user's database table", example=[2])
```

Comment:
```suggestion
    user_id: PositiveInt = Field(..., description="Owner's identifier when registered in the user's database table", example=[2])
```

Reply:
even better ... don't forget to import it! :-)

Reply:
Pedanting pydantics....